src/reducers/versions.tsx

```diff
@@ -354,6 +354,9 @@ export type VersionsState = {
       | null // an error has occured
       | undefined; // data not fetched yet
   };
+  versionInfoLoading: {
+    [versionId: number]: boolean | undefined;
```

Comment:
Ah, good point. I can change the `undefined` case to be `false`. The reason I had the reducer set it to `undefined` was because really it should be deleting the keys but that's not safe in Redux because of time travel.  Setting it to `undefined` would let a hypothetical "garbage collection" process delete the keys safely but YAGNI 🤷‍♂ Setting it to `false` is fine.